### ISSUE TITLE
Setup failed

### ISSUE THREAD

**Issue by belgianrubs** [sdebruyn/homeassistant-bpost-integration#14]
Hello,
First of all, thank you for this integration!
I've set up your newly integration for Home Assistant like stated in the README. My sensor binary_sensor.mail_service_available is always turned off, so went into the debug logs. Everything seemed fine, except for the bPost API response which I don't understand completely.
This is what I get as response:
 `Received response {'refId': 'id', 'status': 'success', 'response': {'isMMTSubscribed': False, 'userId': 'mailaddress', 'isHoliday': False, 'isMailProcessed': True, 'Active': [], 'History': []}}`
Hope this is helpfull, otherwise I'm able to test out other versions.
Regards

**Comment by sdebruyn** [comment]
According to the API you didn't receive any mail. Do you see any mail in the mobile app with that account?

I'm in doubt about what the MMT means. I'll play around with some settings to see if I can get it to change.

**Comment by kekonn** [comment]
I am looking into this as well: I don't have anything that says "My Mail" in the bpost app.

**Comment by sdebruyn** [comment]
Ah I wasn't aware. I have been using this for more than 2 years now. Thanks, I'll add that to the README.

**Comment by belgianrubs** [comment]
Sorry to comment on this closed issue. Didn't even know there was a difference between My Mail and the normal bPost app which I've got probably. The weirdest thing is that I got an email to verify myself when setting up the integration. So I didn't notice that there was a real difference.
Maybe it's good to know for the future when parcels are added to the integration because that's something everyone could use I guess?
